reactions.json

```diff
@@ -54,15 +54,15 @@
     "AgF": "fluroethane",
     "CH3ONa": "methoxyethane"
   },
-  "bromoethane": {
+  "1-bromoethane": {

```

Comment:
bromoethane should be preferred over 1-bromoethane since it's redundant to write that 1